app/config.js

```diff
@@ -149,17 +149,15 @@ var config = convict({
     default: false,
     env: "ENABLE_DB_MOCKS_COOKIES_ONLY",
   },
-  storageApiHost: {
-    doc: "Host of storage API.",
-    format: "host",
-    default: "http://localhost:8000",
-    env: "STORAGE_API_HOST",
-  },
-  storageApiKey: {
-    // Not yet used in deployments, so no validation function yet.
-    doc: "Api key allowing access to storage api, needs to match what's in storage api env.",
-    default: "_doNotUseThisApiKeyInProduction_",
-    env: "STORAGE_API_KEY",
+  gcsCredentials: {
+    doc: "Google Cloud Storage credentials.",
+    default: "/credentials/credentials.json",
+    env: "GOOGLE_APPLICATION_CREDENTIALS",
+  },
```

Comment:
I'm wondering if this is necessary to provide a default value here, we'd want an alert if the variable is not set

Reply:
You'd want to validate that the file exists I guess, this could be left as the default.

Reply:
I don't see this config item being used in the code below

Reply:
It's being used implicitely by the `@google-cloud/storage` package

Reply:
can we add that to the docs?